include/caf/sensor_sampler.h

```diff
@@ -22,8 +42,9 @@ struct sensor_config {
 	const char *dev_name;
 	const char *event_descr;
 	const struct sampled_channel *chans;
-	unsigned int sampling_period_ms;
 	uint8_t chan_cnt;
+	unsigned int sampling_period_ms;
```

Comment:
why moving `sampling_period_ms`?

Reply:
Actually this is moving `uint8_t chan_cnt;` up. Its more natural for me that table pointer is followed by the table size. that is cosmetic cleanup i can drop, I know its not a part of the PR. One of the reasons for this change is my editor was highlighting warning in `sensor_sampler_def` where it was in different order than here... Yes I know its silly.

Reply:
It's not that. I was just curious. In this case it has little difference but in general moving struct members that are uint8/16 often affects padding.

Reply:
I forgot about the padding. I can bring this back, its not a big deal.